src/ui/server/helpers/page-variables/core/insurance/index.ts

```diff
@@ -1,27 +1,23 @@
-import { PRODUCT } from '../../../../content-strings';
 import { ROUTES } from '../../../../constants';
 import { CorePageVariablesInitialInput, CorePageVariables } from '../../../../../types';
 import corePageVariables from '..';
 
-const { START: startRoute, FEEDBACK: feedbackRoute } = ROUTES.INSURANCE;
+const { FEEDBACK: feedbackRoute } = ROUTES.INSURANCE;
 
 /**
  * insuranceCorePageVariables
  * Generate page variables required for every insurance page
  * @param {Object} Page content strings bespoke to the page
  * @param {String} Link to the previous page
+ * @param {String} originalUrl for the page user is on
  * @returns {Object} Common page content strings
  */
-const insuranceCorePageVariables = ({ PAGE_CONTENT_STRINGS, BACK_LINK }: CorePageVariablesInitialInput): CorePageVariables => ({
+const insuranceCorePageVariables = ({ PAGE_CONTENT_STRINGS, BACK_LINK, originalUrl }: CorePageVariablesInitialInput): CorePageVariables => ({
   ...corePageVariables({
     PAGE_CONTENT_STRINGS,
-    PRODUCT: {
-      ...PRODUCT,
-      DESCRIPTION: PRODUCT.DESCRIPTION.APPLICATION,
-    },
     BACK_LINK,
-    START_ROUTE: startRoute,
     FEEDBACK_ROUTE: feedbackRoute,
+    originalUrl,
```

Comment:
Feels strange to have this in camel case when all the others are uppercase, should we call it something like `URL`?

Reply:
changed to `ORIGINAL_URL`